### ISSUE TITLE
simavr1.6 Makefile bug checking for avr-gcc installation

### ISSUE THREAD

**Issue by Jim-Carroll** [osx-cross/homebrew-avr#319]
The current version of the simavr1.6 Makefile.common checks if avr-gcc is installed by looking for a folder called `avr-gcc` in the homebrew Cellar. This did not work on my system (macOS Ventura 13.5). Instead, I have two folders in the Cellar called `avr-gcc@9` and `avr-gcc@12`.

To workaround this issue, I manually created a folder called `avr-gcc` in the Cellar. This allowed the installation to proceed and complete successfully.

```
# workaround
mkdir $(brew --prefix)/Cellar/avr-gcc
# now this works
brew install simavr
```

Looks like the new version of the Makefile fixes this by using a glob pattern, so I guess this cask is just outdated. However, the recommendation to `brew install avr-libc` is incorrect. No such package exists. I believe the error message should say `brew install avr-gcc` instead.

https://github.com/buserror/simavr/blob/master/Makefile.common#L69C4-L69C4

**Comment by gatk555** [comment]
Should this not be opened at buserror/simavr?

**Comment by Jim-Carroll** [comment]
No. It has already been fixed upstream. This repo is just using an older version.

https://github.com/buserror/simavr/commit/9f1014a7ae80a0338d10a6152f2fee0e5a6e410d

**Comment by gatk555** [comment]
My mistake! But there is nothing on the surface to indicate that a simavr build is here.

Thanks, G.

**Comment by Jim-Carroll** [comment]
This formula uses simavr v1.6: https://github.com/osx-cross/homebrew-avr/blob/main/Formula/simavr.rb

That version of simavr has the aforementioned bug.

**Comment by kaidokert** [comment]
Should be fixed by #359
